App/PHP_HTML/login sample/style.css

```diff
@@ -0,0 +1,36 @@
+.navbar-inverse .navbar-nav>li>a {
+    color: black;
+}
+
+input#Male {
```

Comment:
Instead of this, just take the inputs with these ids and give them both a common class such as `gender` for example

then have one css like
`input.<chosen class name> {`